### PR TITLE
Services/APT: Add remaining Applet IDs

### PR DESCRIPTION
**Pull request by Dragios**
Added *eShop (tiger)* in `apt.h`

Added the following Applet IDs in `apt.cpp`:
- *AlternateMenu*
- *Camera*
- *FriendList*
- *GameNotes*
- *InternetBrowser*
- *InstructionManual*
- *Notifications*
- *Miiverse*
- *MiiversePost*
- *AmiiboSettings*
- *PnoteApp*, *PnoteApp2*
- *SnoteApp*, *SnoteApp2*
- *Mint*, *Mint2*
- *Extrapad*, *Extrapad2*
- *Memolib*, *Memolib2*

Some region is incomplete due to the fact that it is listed as **?** or **N/A** in *3dbrew*.

Also fix issue #3224 at the same time

---
This change is [Reviewable](https://reviewable.io/reviews/citra-emu/citra/3232)

### CODE REVIEW

File: src/core/hle/service/apt/apt.cpp [hunk 1]
```diff
@@ -97,12 +97,40 @@ static constexpr size_t NumApplets = 29;
...
+     0x400300000C402, 0x400300000DC02, 0x400300000E202, 0x400300000E802},
+    {AppletId::Error, AppletId::Error2, 0x400300000C502, 0x400300000C502, 0x400300000C502,
+     0x400300000C502, 0x400300000CF02, 0x400300000CF02, 0x400300000CF02},
+    {AppletId::Mint, AppletId::Mint2, 0x400300000C602, 0x400300000CE02, 0x400300000D602,
+     0x400300000C602, 0x400300000DD02, 0x400300000E302, 0x400300000E902},
+    {AppletId::Extrapad, AppletId::Extrapad2, 0x400300000CD02, 0x400300000CD02, 0x400300000CD02,
+     0x400300000CD02, 0x400300000D502, 0x400300000D502, 0x400300000D502},
+    {AppletId::Memolib, AppletId::Memolib2, 0x400300000F602, 0x400300000F602, 0x400300000F602,
+     0x400300000F602, 0x0, 0x0, 0x0},
     // TODO(Subv): Fill in the rest of the titleids
```
Comment (BreadFish64) [inline src/core/hle/service/apt/apt.cpp]: Is this comment still necessary?

**Comment by Subv** [comment]
I want to review this more closely, please hold on merging until then

File: src/core/hle/service/apt/apt.cpp [hunk 2]
```diff
@@ -97,12 +97,40 @@ static constexpr size_t NumApplets = 29;
...
+    {AppletId::GameNotes, AppletId::None, 0x4003000008702, 0x4003000009302, 0x4003000009C02,
+     0x4003000008702, 0x400300000A502, 0x400300000AD02, 0x400300000B502},
+    {AppletId::InternetBrowser, AppletId::None, 0x4003000008802, 0x4003000009402, 0x4003000009D02,
+     0x4003000008802, 0x400300000A602, 0x400300000AE02, 0x400300000B602},
+    {AppletId::InstructionManual, AppletId::None, 0x4003000008602, 0x4003000009202, 0x4003000009B02,
+     0x4003000008602, 0x400300000A402, 0x400300000AC02, 0x400300000B402},
+    {AppletId::Notifications, AppletId::None, 0x4003000008E02, 0x4003000009702, 0x400300000A002,
+     0x4003000008E02, 0x400300000A802, 0x400300000B002, 0x400300000B802},
+    {AppletId::Miiverse, AppletId::None, 0x400300000BC02, 0x400300000BD02, 0x400300000BE02,
+     0x400300000BC02, 0x0, 0x0, 0x0},
```
Comment (Subv) [inline src/core/hle/service/apt/apt.cpp]: 0x4003000009E02, 0x4003000009502, 0x400300000B902

File: src/core/hle/service/apt/apt.cpp [hunk 3]
```diff
@@ -97,12 +97,40 @@ static constexpr size_t NumApplets = 29;
...
+    {AppletId::MiiversePost, AppletId::None, 0x4003000008302, 0x4003000008B02, 0x400300000BA02,
```
Comment (Subv) [inline src/core/hle/service/apt/apt.cpp]: My NS dump from fw 4.5 says all of these are 0x400300000BA02. I'd say use this value until we reverse a newer version.

File: src/core/hle/service/apt/apt.cpp [hunk 4]
```diff
@@ -97,12 +97,40 @@ static constexpr size_t NumApplets = 29;
...
+     0x4003000008302, 0x0, 0x0, 0x0},
+    {AppletId::AmiiboSettings, AppletId::None, 0x4003000009502, 0x4003000009E02, 0x400300000B902,
```
Comment (Subv) [inline src/core/hle/service/apt/apt.cpp]: My NS dump is from 4.5 so i don't have this applet there, i can't verify these values. Judging by how every other value seems to be fine so far, i think this is ok.

File: src/core/hle/service/apt/apt.cpp (hunk 1 above)
Comment (Subv) [reply src/core/hle/service/apt/apt.cpp]: yes

File: src/core/hle/service/apt/apt.cpp [hunk 5]
```diff
@@ -97,12 +97,40 @@ static constexpr size_t NumApplets = 29;
...
+    {AppletId::SnoteApp, AppletId::SnoteApp2, 0x400300000C402, 0x400300000CC02, 0x400300000D402,
+     0x400300000C402, 0x400300000DC02, 0x400300000E202, 0x400300000E802},
+    {AppletId::Error, AppletId::Error2, 0x400300000C502, 0x400300000C502, 0x400300000C502,
+     0x400300000C502, 0x400300000CF02, 0x400300000CF02, 0x400300000CF02},
+    {AppletId::Mint, AppletId::Mint2, 0x400300000C602, 0x400300000CE02, 0x400300000D602,
+     0x400300000C602, 0x400300000DD02, 0x400300000E302, 0x400300000E902},
+    {AppletId::Extrapad, AppletId::Extrapad2, 0x400300000CD02, 0x400300000CD02, 0x400300000CD02,
+     0x400300000CD02, 0x400300000D502, 0x400300000D502, 0x400300000D502},
+    {AppletId::Memolib, AppletId::Memolib2, 0x400300000F602, 0x400300000F602, 0x400300000F602,
+     0x400300000F602, 0x0, 0x0, 0x0},
```
Comment (Subv) [inline src/core/hle/service/apt/apt.cpp]: All of them are 0x400300000F602

File: src/core/hle/service/apt/apt.cpp [hunk 6]
```diff
@@ -97,12 +97,40 @@ static constexpr size_t NumApplets = 29;
...
-    {AppletId::Error, AppletId::Error2, 0x400300000C502, 0x400300000C502, 0x400300000C502,
-     0x400300000C502, 0x400300000CF02, 0x400300000CF02, 0x400300000CF02},
     {AppletId::Ed1, AppletId::Ed2, 0x400300000C102, 0x400300000C902, 0x400300000D102,
      0x400300000C102, 0x400300000D902, 0x400300000DF02, 0x400300000E502},
+    {AppletId::PnoteApp, AppletId::PnoteApp2, 0x400300000C302, 0x400300000CB02, 0x400300000D302,
+     0x400300000C302, 0x400300000DB02, 0x400300000E102, 0x400300000E702},
+    {AppletId::SnoteApp, AppletId::SnoteApp2, 0x400300000C402, 0x400300000CC02, 0x400300000D402,
+     0x400300000C402, 0x400300000DC02, 0x400300000E202, 0x400300000E802},
+    {AppletId::Error, AppletId::Error2, 0x400300000C502, 0x400300000C502, 0x400300000C502,
+     0x400300000C502, 0x400300000CF02, 0x400300000CF02, 0x400300000CF02},
```
Comment (Dragios) [inline src/core/hle/service/apt/apt.cpp]: I'm not sure if this is the correct value as *3dbrew* links it to Fatal error viewer (ErrDisp) which has `00008A02 ` as the region code instead.

Comment (Subv) [reply src/core/hle/service/apt/apt.cpp]: This is correct.

File: src/core/hle/service/apt/apt.cpp [hunk 7]
```diff
@@ -97,12 +97,44 @@ static constexpr size_t NumApplets = 29;
...
+    {AppletId::InternetBrowser, AppletId::None, 0x4003000008802, 0x4003000009402, 0x4003000009D02,
+     0x4003000008802, 0x400300000A602, 0x400300000AE02, 0x400300000B602},
+    {AppletId::InstructionManual, AppletId::None, 0x4003000008602, 0x4003000009202, 0x4003000009B02,
+     0x4003000008602, 0x400300000A402, 0x400300000AC02, 0x400300000B402},
+    {AppletId::Notifications, AppletId::None, 0x4003000008E02, 0x4003000009702, 0x400300000A002,
+     0x4003000008E02, 0x400300000A802, 0x400300000B002, 0x400300000B802},
+    {AppletId::Miiverse, AppletId::None, 0x400300000BC02, 0x400300000BD02, 0x400300000BE02,
+     0x400300000BC02, 0x4003000009E02, 0x4003000009502, 0x400300000B902},
+    {AppletId::MiiversePost, AppletId::None, 0x400300000BA02, 0x400300000BA02, 0x400300000BA02,
+     0x400300000BA02, 0x400300000BA02, 0x400300000BA02, 0x400300000BA02}, // for older version
```
Comment (Subv) [inline src/core/hle/service/apt/apt.cpp]: Please leave a more detailed comment.

**Pull request merged**